cacti/tests/conftest.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
```

Comment:
PEP8 nitpick, imports should be standard library (os, copy), blank line, third party modules (docker, pytest) blank line and finally local imports.